core/backend/src/ChangeSummaryManager.ts

```diff
@@ -183,19 +183,19 @@ export class ChangeSummaryManager {
   private static getExtendedSchemaPath(): string { return path.join(KnownLocations.packageAssetsDir, "IModelChange.02.00.00.ecschema.xml"); }
 
   private static isSummaryAlreadyExtracted(changesFile: ECDb, changeSetId: GuidString): Id64String | undefined {
-    return changesFile.withPreparedStatement("SELECT Summary.Id summaryid FROM imodelchange.ChangeSet WHERE WsgId=?",
-      (stmt: ECSqlStatement) => {
-        stmt.bindString(1, changeSetId);
-        if (DbResult.BE_SQLITE_ROW === stmt.step())
-          return stmt.getValue(0).getId();
+    // eslint-disable-next-line @typescript-eslint/no-deprecated
```

Comment:
Why ignore deprecation warnings instead of updating to use preferred replacement APIs? (Here and lots of other places).

Reply:
I have the same concerns, but the blast radius here is quite a lot, so I'm ok if we are temporarily suppressing the deprecation warnings internally. But I would hope and require we use the new api in our documentation with this pr, and have examples (in example-code snippets)

Reply:
As I understand, we do not offer a sync replacement for the deprecated API, so to switch to ConcurrentQuery in a clean way we may have to make the caller methods async as well. Which would increase the blast radius even more.

Reply:
Is there some reason to believe the people consuming the deprecated APIs will not experience similar friction in attempting to address the new deprecation warnings in their own code? It sounds like our replacement methods are insufficient.

Reply:
@pmconne can you resolve this conversation as you already approved.